lib/node/rpc.js

```diff
@@ -2152,16 +2152,22 @@ class RPC extends RPCBase {
   }
 
   async estimateFee(args, help) {
-    if (help || args.length !== 1)
-      throw new RPCError(errs.MISC_ERROR, 'estimatefee nblocks');
+    if (help || args.length < 1 || args.length > 2)
```

Comment:
We are discussing the return values of `estimateFee` in bcoin right now: https://github.com/bcoin-org/bcoin/pull/889

It's worth noting that Bitcoin Core returns a JSON object for this PRC: https://chainquery.com/bitcoin-cli/estimatesmartfee

And my opinion is that for hsd, we should also be returning a JSON object -- with all the covenant types listed. (And maybe either remove the `type` argument, or add a NEW API or RPC call that specifies `type`?)

I think this may be the most useful to enterprise hsd full nodes, especially given the possibility that covenant types can be combined and batched by large businesses:

```
{
  blocks: 2,
  rate: {
    NONE: 10000,
    OPEN: 15000,
    RENEW: 17589,
    UPDATE: 23000
}
```

Reply:
That return value looks better. I'm not sure what `hsd`'s policy is on breaking changes, but I'm happy to change the signature in this PR if that change wouldn't make this PR harder to merge.